test/views/coronavirus_local_restrictions/results_html_test.rb

```diff
@@ -0,0 +1,117 @@
+require "test_helper"
+
+module CoronavirusLocalRestrictions
+  class ResultsHtmlTest < ActionView::TestCase
+    include CoronavirusLocalRestrictionsHelpers
+
+    describe "current restrictions" do
+      test "rendering tier 4 results for a postcode in tier 4" do
+        render_results_view(local_restriction: { current_alert_level: 4 })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_pretext")
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_four.heading_tier_label")
+      end
+
+      test "rendering tier 3 results for a postcode in tier 3" do
+        render_results_view(local_restriction: { current_alert_level: 3 })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_three.heading_pretext")
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_three.heading_tier_label")
+      end
+
+      test "rendering tier 2 results for a postcode in tier 2" do
+        render_results_view(local_restriction: { current_alert_level: 2 })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_pretext")
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_two.heading_tier_label")
+      end
+
+      test "rendering tier 1 results for a postcode in tier 1" do
+        render_results_view(local_restriction: { current_alert_level: 1 })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_one.heading_pretext")
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.level_one.heading_tier_label")
+      end
+    end
+
+    describe "devolved nations" do
+      test "rendering results for a Welsh postcode" do
+        render_results_view(local_restriction: { country_name: "Wales" })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.devolved_nations.wales.guidance.label")
+      end
+
+      test "rendering results for a Scottish postcode" do
+        render_results_view(local_restriction: { country_name: "Scotland" })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.devolved_nations.scotland.guidance.label")
+      end
+
+      test "rendering results for a Northern Irish postcode" do
+        render_results_view(local_restriction: { country_name: "Northern Ireland" })
+
+        assert_includes rendered, I18n.t("coronavirus_local_restrictions.results.devolved_nations.northern_ireland.guidance.label")
+      end
+    end
+
+    describe "future restrictions" do
+      area = "Naboo"
```

Comment:
It feels a little unusual to define a variable like this.

I think the convention for minitest would be to set an ivar in a before block. For example: `before { @area = "Naboo" }`

Reply:
thanks, have updated.